docs/learn-legitimate-projects.md

```diff
@@ -1,242 +1,162 @@
+
+Many parachain candidates offer a way to participate through their site as well. But you should ensure that they are using the crowdloan pallet in the background, and they're simply wrapping that in a nicer, more user-friendly UI. If instead, their crowdloan interface *transfers* funds to an account, these funds will be totally under their control, and this means you need to fully trust that the team will use the funds for the crowdloan, return your share to you when the lease period ends or if they don’t get a spot, and will secure them properly! And if they're doing something like that, it should be explicitly mentioned in their site and documentation.
+
+That being said, some teams have been doing token sales or swaps in an attempt to get a head start in raising funds for the auctions, but these are not crowdloans and still require full trust in the team. So, if you plan to participate in these token swaps, make sure the project is reputable and that you're getting the correct information through their official site and social media channels.
+
+Similarly, several centralised exchanges are creating ways to participate in crowdloans through their platforms, some wallets are integrating crowdloan functionality to their apps, and more are sure to follow. Any legitimate effort should be using the native crowdloans module in the background and offering a more streamlined user experience. Making sure that's the case is necessary before using these services, but in any case, it still involves trusting the exchange or the service provider.
+
+### Additional material
+If you're interested in learning more about how to check facts and verify claims, have a look at the following material. Fact-checking is a skill necessary not only for DYOR but for filtering out the ton of information that we come across daily on the internet.
+
+1. A very nice YouTube series on the art of fact-checking that covers a lot of ground can be found [here](https://www.youtube.com/watch?v=L4aNmdL3Hr0&list=PL8dPuuaLjXtN07XYqqWSKpPrtNDiCHTzU).
+2. Another great resource on fact-checking, for those who prefer to read, can be found [here](https://webliteracy.pressbooks.com).
+3. Wikipedia article on [fact-checking](https://en.wikipedia.org/wiki/Fact-checking)
+
+Finally, you should also check our complementary guide on [how to identify scams](learn-scams). It's not about projects but how to identify outright scams and avoid them, as well as how to protect your sensitive information.
```

Comment:
"learn-scams.md" -> we added .md to reference pages now after updating the site to use docusaurus after getting some feedback

Reply:
So, all links referencing pages on the wiki need the .md?

Reply:
I changed them all to add the .md extension